insert.go

```diff
@@ -32,22 +32,15 @@ func (b *InsertStmt) Build(d Dialect, buf Buffer) error {
 	buf.WriteString(d.QuoteIdent(b.Table))
 
 	buf.WriteString(" (")
-
-	placeholder := new(bytes.Buffer)
-	placeholder.WriteRune('(')
 	for i, col := range b.Column {
 		if i > 0 {
 			buf.WriteString(",")
-			placeholder.WriteString(",")
 		}
 		buf.WriteString(d.QuoteIdent(col))
-		placeholder.WriteString(d.Placeholder())
 	}
-	placeholder.WriteString(")")
-	placeholderStr := placeholder.String()
-
 	buf.WriteString(") VALUES ")
 
+	placeholderStr := "(" + strings.Join(strings.Split(strings.Repeat(placeholder, len(b.Column)), ""), ",") + ")"
```

Comment:
@taylorchu we should use byte buffers as much as possible to reduce allocs. I think this change definitely needs to be benchmarked before/after.